### ISSUE TITLE
Integration not found

### ISSUE THREAD

**Issue by elsayed-atef** [issc29/probot-prevent-public-repos#4]
i created my stand alone deployment for Github Enterprise . i specified the parameter "GHE_HOST" to my github enterprise and gave the permission for Repository administration - Read & Write and Single File - Read-only
 Path: .github/prevent-public-repos.yml

i also created a private repo "org-settings" and added the file  ".github/prevent-public-repos.yml". I got the following error

```
10:05:32.136Z ERROR Probot:
  Promise {
    <rejected> { [Error: {"message":"Integration not found","documentation_url":"https://developer.github.com/v3"}]
    message: '{"message":"Integration not found","documentation_url":"https://developer.github.com/v3"}',
    code: 404,
    status: 'Not Found',
    headers:
     { server: 'GitHub.com',
       date: 'Tue, 14 Aug 2018 10:05:32 GMT',
       'content-type': 'application/json; charset=utf-8',
       'content-length': '89',
       connection: 'close',
       status: '404 Not Found',
       'x-github-media-type': 'github.machine-man-preview; format=json',
       'access-control-expose-headers': 'ETag, Link, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval',
       'access-control-allow-origin': '*',
       'strict-transport-security': 'max-age=31536000; includeSubdomains; preload',
       'x-frame-options': 'deny',
       'x-content-type-options': 'nosniff',
       'x-xss-protection': '1; mode=block',
       'referrer-policy': 'origin-when-cross-origin, strict-origin-when-cross-origin',
       'content-security-policy': 'default-src \'none\'',
       'x-runtime-rack': '0.015388',
       'x-github-request-id': '7887:4D0A:127AFC4:2612784:5B72A96B' } } }
  --
  err: {
    "code": 404,
    "status": "Not Found",
    "message": "{\"message\":\"Integration not found\",\"documentation_url\":\"https://developer.github.com/v3\"}"
  }
```

**Comment by issc29** [comment]
@elsayed-atef it sounds like the App probably was not configured properly, did you follow the [Deployment Article](https://probot.github.io/docs/deployment/) and update the [App.json](https://github.com/issc29/probot-prevent-public-repos/blob/master/app.json) file?

Additionally you can reach out to your GitHub Sales/Support contacts to debug further (and mention me).

**Comment by elsayed-atef** [comment]
@issc29 i created a `.env` file , but what is the format for App.json

**Comment by issc29** [comment]
@elsayed-atef sorry I meant the `.env` not App.json

after further research it looks like Probot implemented GHE support in v4.0.0 of Probot, but this app is using v3.0.0. I would recommend updating the Probot version to at least 4.0.0 and then try deploying. 

https://github.com/issc29/probot-prevent-public-repos/blob/b718e5861c166c3e8819df3e8de917c330026a8a/package.json#L12-L14

I will also work on a PR to upgrade the Probot version for this package.

**Comment by elsayed-atef** [comment]
@issc29 Thanks , I used these and now it's working 
```
"dependencies": {
    "probot": "^7.2.0",
    "probot-config": "^0.2.0"
  },
```
